### ISSUE TITLE
Can you change number of LEDs after initial WiFI setup?

### ISSUE THREAD

**Issue by tmur11** [HomeKidd/Homekit-WS2812B-controller#57]
Can you change number of LEDs after the initial WiFI setup?

**Comment by fmland2** [comment]
> Can you change number of LEDs after the initial WiFI setup?

I would love to know this as well.

**Comment by andywaplinger** [comment]
No, you must reset the controller by holding the button for 10 seconds and re-enter the wifi setup flow.
